specification/logs/semantic_conventions/Instrumentation/mobile-events.md

```diff
@@ -0,0 +1,59 @@
+# Semantic Conventions for mobile instrumentation
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for instrumentation on mobile platforms.
+
+<!-- toc -->
+
+- [Lifecycle instrumentation](#lifecycle-instrumentation)
+  * [`state` values](#state-values)
+    + [iOS](#ios)
+    + [Android](#android)
+
+<!-- tocstop -->
+
+## Lifecycle instrumentation
+
+This section defines how to apply semantic conventions when instrumenting application lifecycle.
+This event is meant to be used in conjunction with `os.name` [resource semantic convention](https://github.com/open-telemetry/semantic-conventions/blob/main/specification/resource/semantic_conventions/os.md) to identify platform.
+
+**Event name**: `client.lifecycle`
+
+**Event domain**: `device`
+
+The following attributes are stored in the `event.data` map.
+
+| Attribute | Type   | Description                                | Examples          | Requirement level |
+|-----------|--------|--------------------------------------------|-------------------|-------------------|
+| `state`   | String | The state entered at the time of the event | platform specific | Required          |
+
+### `state` values
+
+#### iOS
+
+The iOS lifecycle states are defined in the [UIApplicationDelegate documentation](https://developer.apple.com/documentation/uikit/uiapplicationdelegate#1656902),
+and from which the `OS terminology` column values are derived.
+
+| Name         | OS terminology                 | description                               |
+|--------------|--------------------------------|-------------------------------------------|
+| `active`     | applicationDidBecomeActive     | The app has become "active"               |
+| `inactive`   | applicationWillResignActive    | The app is about to become "inactive".    |
+| `background` | applicationDidEnterBackground  | The app is now in the background.         |
+| `foreground` | applicationWillEnterForeground | The app is about to enter the foreground. |
+| `terminate`  | applicationWillTerminate       | The app is about to terminate.            |
+
+#### Android
+
+The Android lifecycle states are defined in [Activty lifecycle callbacks](https://developer.android.com/guide/components/activities/activity-lifecycle#lc), and from which the `OS idenfifiers` are derived.
+
+| Name         | OS terminology | description                                       |
+|--------------|----------------|---------------------------------------------------|
+| `created`    | [1]            | The app has been launched.                        |
+| `foreground` | [2]            | The app is in the foreground or otherwise active. |
+| `background` | [3]            | The app is in the background.                     |
+
+
+[1] Any time before Activity.onResume() or, if the app has no Activity, Context.startService() has been called in the app for the first time
```

Comment:
I had some DM conversations with @LikeTheSalad and we decided that given the inconsistency and messiness of Android's app start lifecycle, we are going to shrink the definition of `created` to simply the `Application` object has been fully created - this takes care of the case of background app starts that don't create the UI (e.g. to handle a notification or run a scheduled job), and the app will flip between background and foreground until the process is killed. 

This definition is perhaps less useful, but it's more consistent and works better with Android's ill-defined app lifecycle. When it comes to the implementation, we'll have to decided some nuances, but that's not the concern of the semantic convention 😅